src/Sulu/Bundle/RouteBundle/Resources/views/Template/content-types/route.html.twig

```diff
@@ -5,7 +5,7 @@
      data-form="true"
      data-type="resourceLocator"
      data-aura-component="resource-locator@sulucontent"
-     data-aura-history-api="/admin/api/routes?history=true&entityClass=<%= entityClass %>&entityId=<%= entityId %>&locale={{ languageCode }}"
+     data-aura-history-api="<% if(typeof entityClass !== 'undefined') { %>/admin/api/routes?history=true&entityClass=<%= entityClass %>&entityId=<%= entityId %>&locale={{ languageCode }}<% } %>"
```

Comment:
the "older" versions and also the current "0.5.1" does not pass this parameter to the form template and breaks.

The feature which was introduced by the PR #3315 is optional and the javascript will not show the button when the URL is not there.